runners/direct-java/pom.xml

```diff
@@ -83,7 +91,7 @@
               <failIfNoTests>true</failIfNoTests>
               <dependenciesToScan>
                 <dependency>org.apache.beam:beam-sdks-java-core</dependency>
-                <dependency>org.apache.beam:beam-runners-java-core</dependency>
+                <dependency>org.apache.beam:beam-runners-core-java</dependency>
```

Comment:
Hmm, I'll take a look in a bit. It might be that the reorganized deps don't work well with `dependenciesToScan`.